deploy/deployment-manager/compute_engine.py

```diff
@@ -52,7 +51,7 @@ def generate_config(context):
             ),
             "serviceAccounts": [
                 {
-                    "email": f"$(ref.{sa_name}.email)",
+                    "email": get_service_account_email(sa_name, project),
```

Comment:
we can't use `ref` because we might be using an already-deployed service account